### PR TITLE
Revise audio document

### PR DESCRIPTION
**Pull request by sangchul1011**
Revise audio document

The summary is revised in audio.md.
sound.md is renamed to sound-manager.md for Sound-Manager API.

### CODE REVIEW

**Comment by sangchul1011** [comment]
Please review this patch. Thanks.

File: docs/application/native/guides/multimedia/audio.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Audio Management
 
 
-You can control the audio behavior of the application. You can control the output volume and sound session types, and query various information from sound devices. You can also use OpenAL to specify the objects and operations in producing high-quality audio output.
+You can control the audio behavior of the application. You can control the output volume and sound conflict scenario, and query various information from sound devices. You can also use OpenAL to specify the objects and operations in producing low-latency audio output.
```
Comment (divyk) [inline docs/application/native/guides/multimedia/audio.md]: You can control the output volume and sound conflict scenarios. You can query various information from sound devices.

Please clarify, the above sentence can be rewritten in the following style also:
- You can also query various information about the output from the sound devices.

please correct my understanding, I wrote this with my limited knowledge.

Comment (sangchul1011) [reply docs/application/native/guides/multimedia/audio.md]: I can rewrite the sentence like the style you said
 - You can also query..

but I think various information "about the output" is not proper.
The information is consist of device type, name, id, and so on.
I'll update it soon again.

File: docs/application/native/guides/multimedia/audio.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ You can use the following audio management features in your native applications:
 
 - [Sound Manager](sound-manager.md)
 
-  You can control the output volume of a specific volume type (aka sound type). You can define which stream type is used for a playback or recording handle in your application, and manage sound conflict senario by using stream focus. You can also query for various information related to the connected sound devices.
+  You can control the output volume of a specific volume type (aka sound type). You can define which stream type is used for a playback or recording handle in your application, and manage sound conflict senarios by using stream focus. You can also query for various information related to the connected sound devices, e.g. type, name, id, direction and state.
```
Comment (divyk) [inline docs/application/native/guides/multimedia/audio.md]: for example, type, name, id, direction and state.

Comment (sangchul1011) [reply docs/application/native/guides/multimedia/audio.md]: ok, I'll update it soon.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**